### ISSUE TITLE
A11y_ML .NET Model Builder_ Scenarios_ Info and Relationships: Proper control type is not provided for "Feedback" control

### ISSUE THREAD

**Issue by SaisreeDandamudi** [dotnet/machinelearning-modelbuilder#1572]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #BM-VisualStudioClient-Win32-Jul21; #A11ySev2; #Visual Studio; #WCAG1.3.1; 

### Environment Details:
Microsoft Visual Studio Enterprise 2022 Preview Version 17.0.0 Preview 1.1

### Repro Steps:

1. Open Visual Studio Enterprise 2022 Preview
2. Create a new .Net Core Console App (C#)
3. Tab to "Solution Explorer" and right click on the project (Shift + F10) and then from the expanded list Select "Add" and then from the list items select "Machine learning"
4. Tab till "Feedback" control and verify the control type for the feedback control using Accessibility Insights for desktop

### Actual Result:
Proper control type is not provided for "Feedback" control. Control type is given as "Text"

### Expected Result:
Since feedback is an actionable control control type should not be "Text". Proper control type should be provided for the control

### User Impact:
Screen Reader users will get confused if proper control type is not given for the controls

**Comment by zewditu** [comment]
@SaisreeDandamudi the narrator says  "Feedback", are we expected to say " Feedback button.." or are we expect it to say " feedback ......any actionable control"?

**Comment by NiharikaTipirishetty** [comment]
@zewditu the role of 'Feedback' control should be defined, role of feedback should defined as button.

**Comment by beccamc** [comment]
Validate on  rel/candidate build 16.7.2.2141903. See issue 1627 for instructions for install.

**Comment by NiharikaTipirishetty** [comment]
Issue is fixed. Control type for feedback control is defined as button. Issue is fixed
![1572_Narrator_fixed](https://user-images.githubusercontent.com/83570638/130208484-bcb60c1f-d523-4ab8-a5f1-c188fb24a462.png)
![1572_NVDA_fixed](https://user-images.githubusercontent.com/83570638/130208490-b862d9aa-7646-4936-9e64-79df3b899da6.png)

**Comment by akashelijah05** [comment]
#Closed;
